locales/en.default.json

```diff
@@ -637,7 +755,50 @@
             "active": "Expires on {{ expiry }}",
             "redeem": "Use this code at checkout to redeem your gift card",
             "shop_link": "Start shopping",
-            "print": "Print"
+            "left": "left"
+        }
+    },
+    "popup": {
+        "error": {
+            "title": "Error",
+            "ok": "Ok"
+        },
+        "guest": {
+            "title": "Sign in to your account",
+            "check_out_faster": "You'll be able to check out faster",
+            "email": "Email address",
+            "password": "Password",
+            "forgot_pass": "Forgot password?",
+            "sign_in": "Sign in",
+            "without_sign_in": "Continue without signing in",
+            "after_purchasse": "You'll have the option to register for an account after your purchase.",
+            "register_mow": "register for an account now",
+            "as_guest": "Continue as a guest"
+        }
+
+    },
+    "confirmation": {
+        "confirm_email": "Thank you for confirming your email",
+        "click_login_link": "click here to log in",
+        "pass_recovery": "Password recovery",
+        "check_email_to_reset_pass": "Please check your email to reset your password.",
```

Comment:
pass → password

Reply:
@asvishnyakov for consistency it should be "check_email_to_reset_password" then

Reply:
@aartiomov @n2pro yes, I mean abbreviations should not be used